cores/esp8266/Stream.cpp

```diff
@@ -288,6 +289,59 @@ String Stream::readStringUntil(const char* terminator, uint32_t untilTotalNumber
     return ret;
 }
 
+String Stream::readStreamString(const ssize_t maxLen ,const oneShotMs::timeType timeoutMs) {
```

Comment:
This is exactly the same as [::sendSize](https://github.com/esp8266/Arduino/blob/0301465a3c53fb42986814f81532b512d47d37fd/cores/esp8266/Stream.h#L212-L213).

Reply:
Not Exactly, It constructs a String Object and a Stream Object attached to it, then populate it, then returns the String Object, it's behavior is more like [::readString](https://github.com/esp8266/Arduino/blob/0301465a3c53fb42986814f81532b512d47d37fd/cores/esp8266/Stream.h#L116) 

`sendSize' expect you to pass a Pointer to a Stream and returns the total number of bytes.

`readStreamString` is just a wrapper for the String and Stream contruction.